### PR TITLE
SCRUM-41.2: Added 'manager_id' to Faculty.java. Created API for creating faculty and submission period.

### PR DESCRIPTION
**Pull request by peguin818**
@Alanle1011 check thoroughly and carefully

### CODE REVIEW

File: src/main/java/com/example/backend1640/entity/Faculty.java [hunk 1]
```diff
@@ -22,6 +22,10 @@ public class Faculty {
     @Column(name = "FACULTY_NAME", nullable = false)
     private String faculty_name;
 
+    @OneToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "ID", insertable=false, updatable=false)
```
Comment (Alanle1011) [inline src/main/java/com/example/backend1640/entity/Faculty.java]: Ko cần đâu @Column là đc

File: src/main/java/com/example/backend1640/dto/FacultyDTO.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class FacultyDTO {
+    private String faculty_name;
+
+    //Don't know to add manager_id or not
```
Comment (Alanle1011) [inline src/main/java/com/example/backend1640/dto/FacultyDTO.java]: Ko cần đâu

**Comment by peguin818** [comment]
@Alanle1011 tested creating submission period. Worked as intended
image

**Comment by peguin818** [comment]
@Alanle1011 coi giúp bố dm

**Review by Alanle1011 — APPROVED** [review]
LGTM

**Pull request merged**